src/IssueViz.Security.UnitTests/Hotspots/HotspotMatcherTests.cs

```diff
@@ -30,6 +36,188 @@ public class HotspotMatcherTests
     [TestMethod]
     public void MefCtor_CheckExports()
     {
-        MefTestHelpers.CheckTypeCanBeImported<NoOpHotspotMatcher, IHotspotMatcher>();
+        MefTestHelpers.CheckTypeCanBeImported<HotspotMatcher, IHotspotMatcher>();
+    }
```

Comment:
FYI we now have additional test helpers for [CheckIsNonSharedMefComponent](https://github.com/search?q=repo%3ASonarSource%2Fsonarlint-visualstudio+CheckIsNonSharedMefComponent&type=code) and [CheckIsSingletonMefComponent](https://github.com/search?q=repo%3ASonarSource%2Fsonarlint-visualstudio%20CheckIsSingletonMefComponent&type=code).

When you touch a test file that doesn't have a `CheckIsNonSharedMefComponent` / `CheckIsSingletonMefComponent`, could you add one please?